ui/tests/acceptance/pki/pki-engine-workflow-test.js

```diff
@@ -341,7 +273,6 @@ module('Acceptance | pki workflow', function (hooks) {
         `/vault/secrets/${this.mountPath}/pki/keys/${keyId}/details`,
         'navigates to details after save'
       );
-      await this.pauseTest;
```

Comment:
welp, how did that stay in there? I guess it's not `this.pauseTest();`

Reply:
Aha! I've noticed this a couple passthroughs and been meaning to take out! Thanks @hashishaw !!